src/main/scala/org/zalando/react/nakadi/ReactiveNakadi.scala

```diff
@@ -86,6 +96,66 @@ class ReactiveNakadi {
     consumeWithOffsetSink(props, ReactiveNakadi.ConsumerDefaultDispatcher)
   }
 
+  def producerFlow(producerProperties: ProducerProperties)(implicit system: ActorSystem): Flow[ProducerMessage, Seq[String], NotUsed] = {
+    import org.zalando.react.nakadi.client.models.JsonOps._
+
+
+    import akka.http.scaladsl.model.MediaTypes._
+    import akka.http.scaladsl.model.headers._
+    import concurrent.duration._
+
+    implicit val materializer = ActorMaterializer()
+
+    val superPool = Http().superPool[Seq[String]]()
```

Comment:
There is already a client implementation in `src/main/scala/org/zalando/react/nakadi/client/NakadiClientImpl.scala`. Perhaps it would be more suitable to use/extend that. I could be wrong, but I had issues with connecting to invalidated SSL endpoints, this may run into the same problem. At the very least by using the already existing implementation it removed duplicated code. (I am aware that the current client implementation could probably be improved / cleaned up a bit 😄  )